ludwig/modules/loss_modules.py

```diff
@@ -165,6 +165,26 @@ def forward(self, preds: Tensor, target: Tensor) -> Tensor:
         return self.loss_fn(preds, target)
 
 
+@register_loss("sequence_softmax_cross_entropy", [SEQUENCE, TEXT])
```

Comment:
nit: There were a few other losses specific to seq decoders. Are they planned foe a separate PR?